index.html

```diff
@@ -0,0 +1,304 @@
+    </div>
+       </div>
+       <div class="card">
+         <div class="project-img"></div>
+         <div class="text">
+          <h3>Mutli post stories</h2>
+          <h4>Gain + Glory</h4>
+          <section class = "tags">
+          <li class="tag">Ruby on Rails</li>
+            <li class="tag">Css</li>
+            <li class="tag">Javascript</li>
+            <li class="tag">Html</li>
+          </section>
+          <section class="project">
+      <button class="project">See Project</button>
+   </section> 
+  </div>
+  </div>
+       </div>
+      </section>
+        </section>
+<section class="about-me">
+        <div class="about-myself">
+          <div class="bg-images">
+          <img class="img1" src="./images/frame5.png" >
+          <img class="img2" src="./images/frame4.png">
+          </div>
+          <div class="content">
+          <h1 >About Me</h1>
+          <p class="support-text">
+             I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+          </p>
+           <button class="project">Get my Resume</button>
+            </div>
+          </div >
+         <section class= "grids">
+         <div class="grid">
+            <img class="grid-image" src="./images/language.png" alt="">
+            <div class="txt1">
+            <h3 class="lang-title">Languages</h3>
+             <section class = "tags1">
+            <li class="tag1">Javascript</li>
+            <li class="tag1">Ruby</li>
+            <li class="tag1">Html</li>
+            <li class="tag1">CSS</li>
+          </section>
+          </div>
+         </div>
+          <div class="grid">
+          <img  class="grid-image" src="./images/frameworks.png" alt="">
+          <div class="txt1">
+          <h3>Frameworks</h3>
+           <div class = "tags1">
+            <li class="tag1">Bootstrap</li>
+            <li class="tag1">Ruby on Rails</li>
+            <li class="tag1">Rspec</li>
+           </div>
+             <div class = "tags2">
+            <li class="tag1">Capybara</li>
+            <li class="tag1">Selenium</li>
+          </div>
+        </div>
+          </div>
+       <div class="grid">
+        <img class="grid-image"src="./images/skills.png" alt="">
+        <div class="txt1">
+        <h3>Skills</h3>
+        <div class = "tags1">
+           <li class="tag1">Codekit</li>
+            <li class="tag1">Github</li>
+            <li class="tag1">Codepen</li>
+            <li class="tag1">Gitlab</li>
+  </div>
+            <div class="tags2">
+            <li class="tag1">Terminal</li>
+            </div>
+          </div>
+     </div> 
+         </section>
+    <div class="text">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    </div>
+   <form action="https://formspree.io/f/xpznodvr" method="post">
+  <ul>
```

Comment:
Kudos @mumo-esther for the job well done 👍, but your form spree is still not working yet. The submission is throwing an error.  I will like to suggest that you try this method 👇 and test that your form is submitted before you request another review.
`<form action="https://formspree.io/f/xpznodvr" method="post" id="form">`